packages/app/lib/ui/screens/species.dart

```diff
@@ -115,6 +116,8 @@ class _SpeciesProfileState extends State<SpeciesProfile> {
           ])),
           RelatedSightings(id: species.id),
           const SliverToBoxAdapter(child: SizedBox(height: 20.0)),
+          SliverToBoxAdapter(child: SpeciesUsesAggregate(id: species.id)),
+          const SliverToBoxAdapter(child: SizedBox(height: 20.0)),
```

Comment:
Looking at the wireframes I believe that the "sightings grid" comes last, then you can move your widget into the `SliverList` which accepts only widgets aka you don't need an adapter then 😅

Reply:
Nice! No adaptor :fist_raised: